htdocs/modules/system/admin/modulesadmin/modulesadmin.php

```
@@ -832,9 +832,7 @@ function xoops_module_update($dirname)
     $dirname = trim((string) $dirname);
     $xoopsDB =& $GLOBALS['xoopsDB'];
 
-    $myts = \MyTextSanitizer::getInstance();
-
-    $dirname        = $myts->htmlSpecialChars(trim($dirname));
+    $dirname        = htmlspecialchars(trim($dirname), ENT_QUOTES | ENT_HTML5, 'UTF-8');
     /** @var XoopsModuleHandler $module_handler */
     $module_handler = xoops_getHandler('module');
```

PHPMD (2.15.0)

[error] The variable $module_handler is not named in camelCase. (undefined)

(CamelCaseVariableName)